ssc/cmod_pvsamv1.cpp

```diff
@@ -969,7 +997,10 @@ static var_info _cm_vtab_pvsamv1[] = {
             { SSC_OUTPUT, SSC_NUMBER, "annual_ac_after_inv_pntloss", "AC output after inverter night tare loss", "kWh", "", "Annual (Year 1)", "", "", "" },
             */
             { SSC_OUTPUT, SSC_NUMBER, "annual_ac_wiring_loss", "AC wiring loss", "kWh", "", "Annual (Year 1)", "", "", "" },
-            { SSC_OUTPUT, SSC_NUMBER, "annual_transmission_loss", "Transmission loss", "kWh", "", "Annual (Year 1)", "", "", "" },
+            { SSC_OUTPUT, SSC_NUMBER, "annual_subhourly_clipping_loss", "Subhourly clipping correction loss", "kWh", "", "Annual (Year 1)", "", "", "" },
+            { SSC_OUTPUT, SSC_NUMBER, "annual_subhourly_clipping_factor", "Subhourly clipping correction loss factor", "", "", "Annual (Year 1)", "", "", "" },
```

Comment:
@cpaulgilman curious on your thoughts if these two annual outputs should be reported if the subhourly clipping loss model isn't enabled. I don't have a strong opinion one way or the other for the annual results.

Reply:
_factor outputs were for debugging and were removed.